### ISSUE TITLE
500 Error from server on beta version 2.11-3-beta of Android App.

### ISSUE THREAD

**Issue by Bryan76** [openhab/openhab-android#1758]
openHAB Android client 2.11.0 correctly talks to my openHAB hab server. Beta 2.11.3-beta reports a 500 error. 

I have openHAB Server 2.5.0-1, Rest API Version 3.

Thoughts?

**Comment by maniac103** [comment]
One thought would be 'why didn't you fill out the issue template, particularly the app and server log?' ;-)

**Comment by Bryan76** [comment]
> One thought would be 'why didn't you fill out the issue template, particularly the app and server log?' ;-)

Odd, I swear when I created the initial report, that template was not in the text box.  I will be able to get that info to you later this week.  My apologies.

**Comment by openhab-5iver** [comment]
We're getting the same...

### Actual behaviour
Describe us what happens

After opening the app, you get a 500 error. All was working fine with previous app version.

![Screenshot_20200106-152542](https://user-images.githubusercontent.com/21967853/71846948-bd2a0a80-3099-11ea-9ff7-8ba909d09ece.png)

### Expected behaviour
Describe us what should happen

No 500 error.
 
### Steps to reproduce
1. Describe us how to reproduce the issue
2. Use a list for it

1. Install 2.11-3-beta
2. Launch the app
4. Use app

When the local configuration is removed, everything seems to work OK without the 500 error. However, it looks like this is another bug, since the app is just using the remote settings that I have configured (no auto discovery). When the remote settings are removed, the app crashes (I sent in a report) and the app data needs to be cleared to launch it again without a crash. After clearing the app data, auto discovery runs and the 500 error returns.

### Can you reproduce the issue in demo mode?



No

### Environment data
#### Client
* Android version: 10
* Device model: Google Pixel XL
* App version : 2.11-3-beta
* Build flavor : PlayStore
* Device language: English

#### Server
* Server version: S1778 (last snapshot before 2.5 stable was released)
* Reverse Proxy: No
* Authentication method : None

#### openHAB Cloud
* Self hosted cloud or myopenhab.org: myopenhab.org
* openHAB Cloud version : 
* Authentication method : 

### Logs
#### App log
<details>
  <summary>Click to expand</summary>

```
-----------------------
Device information
Fingerprint: google/marlin/marlin:10/QP1A.191005.007.A3/5972272:user/release-keys
Model: Pixel XL
Manufacturer: Google
Brand: google
Device: marlin
Product: marlin
OS: 10
-----------------------

--------- beginning of events
01-06 16:05:18.295  3552  3552 I am_on_create_called: [0,org.openhab.habdroid.ui.MainActivity,performCreate]
01-06 16:05:18.353  3552  3552 I am_on_start_called: [0,org.openhab.habdroid.ui.MainActivity,handleStartActivity]
01-06 16:05:18.369  3552  3552 I am_on_resume_called: [0,org.openhab.habdroid.ui.MainActivity,RESUME_ACTIVITY]
01-06 16:05:18.381  3552  3552 I am_on_top_resumed_gained_called: [0,org.openhab.habdroid.ui.MainActivity,topStateChangedWhenResumed]
01-06 16:05:23.956  3552  3552 I am_on_top_resumed_lost_called: [0,org.openhab.habdroid.ui.MainActivity,topStateChangedWhenResumed]
01-06 16:05:23.996  3552  3552 I am_on_paused_called: [0,org.openhab.habdroid.ui.MainActivity,performPause]
01-06 16:05:24.465  3552  3552 I am_on_stop_called: [0,org.openhab.habdroid.ui.MainActivity,STOP_ACTIVITY_ITEM]
01-06 16:32:08.750  3552  3552 I am_on_restart_called: [0,org.openhab.habdroid.ui.MainActivity,performRestartActivity]
--------- beginning of main
01-06 16:32:08.757  3552  3552 D MainActivity: onStart()
01-06 16:32:08.757  3552  3552 D PageConnectionHolderFragment: onStart(), started false
01-06 16:32:08.758  3552  3552 D MainActivity: onAvailableConnectionChanged()
01-06 16:32:08.759  3552  3552 D ContentController: Update to connection null (message null)
01-06 16:32:08.760  3552  3552 D PageConnectionHolderFragment: updateActiveConnections: URL list [], connection null
01-06 16:32:08.775  3552  3552 I am_on_start_called: [0,org.openhab.habdroid.ui.MainActivity,handleStartActivity]
01-06 16:32:08.777  3552  3552 D MainActivity: onNewIntent()
01-06 16:32:08.777  3552  3552 D MainActivity: Got intent: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10600000 cmp=org.openhab.habdroid.beta/org.openhab.habdroid.ui.MainActivity bnds=[48,1123][317,1460] }
01-06 16:32:08.778  3552  3552 D MainActivity: onResume()
01-06 16:32:08.780  3552  3552 I am_on_resume_called: [0,org.openhab.habdroid.ui.MainActivity,RESUME_ACTIVITY]
01-06 16:32:08.780  3552  3552 I am_on_top_resumed_gained_called: [0,org.openhab.habdroid.ui.MainActivity,topWhenResuming]
01-06 16:32:08.782  3552  3552 D MainActivity: onCreateOptionsMenu()
01-06 16:32:08.783  3552  3552 D MainActivity: onPrepareOptionsMenu()
01-06 16:32:08.784  3552  3705 D AbstractConnection: Checking reachability of https://<openhab-local-address>:8443/
01-06 16:32:08.838  3552  3705 D AbstractConnection: Socket connected (attempt  0)
01-06 16:32:08.839  3552  3705 D ConnectionFactory: Connecting to local URL
01-06 16:32:08.839  3552  3552 D MainActivity: onAvailableConnectionChanged()
01-06 16:32:08.843  3552  3552 D ContentController: Update to connection org.openhab.habdroid.core.connection.DefaultConnection@8766f67 (message null)
01-06 16:32:08.844  3552  3552 D PageConnectionHolderFragment: updateActiveConnections: URL list [], connection org.openhab.habdroid.core.connection.DefaultConnection@8766f67
01-06 16:32:08.854  3552  3552 D MainActivity: onCreateOptionsMenu()
01-06 16:32:08.854  3552  3552 D MainActivity: onPrepareOptionsMenu()
01-06 16:32:08.982  3552  3552 E MainActivity: Error: org.openhab.habdroid.util.HttpClient$HttpException: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:08.982  3552  3552 E MainActivity: org.openhab.habdroid.util.HttpClient$HttpException: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:08.982  3552  3552 E MainActivity:  at org.openhab.habdroid.util.HttpClient$method$$inlined$suspendCancellableCoroutine$lambda$1.onFailure(HttpClient.kt:149)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.RealCall$AsyncCall.run(RealCall.kt:146)
01-06 16:32:08.982  3552  3552 E MainActivity:  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
01-06 16:32:08.982  3552  3552 E MainActivity:  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
01-06 16:32:08.982  3552  3552 E MainActivity:  at java.lang.Thread.run(Thread.java:919)
01-06 16:32:08.982  3552  3552 E MainActivity: Caused by: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:08.982  3552  3552 E MainActivity:  at com.android.org.conscrypt.ConscryptFileDescriptorSocket.startHandshake(ConscryptFileDescriptorSocket.java:231)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.kt:350)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.kt:308)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.connection.RealConnection.connect(RealConnection.kt:182)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.kt:238)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.kt:111)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.kt:79)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.connection.Transmitter.newExchange$okhttp(Transmitter.kt:163)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:35)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:82)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:84)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:71)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.logging.HttpLoggingInterceptor.intercept(HttpLoggingInterceptor.kt:152)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.kt:194)
01-06 16:32:08.982  3552  3552 E MainActivity:  at okhttp3.RealCall$AsyncCall.run(RealCall.kt:138)
01-06 16:32:08.982  3552  3552 E MainActivity:  ... 3 more
01-06 16:32:08.982  3552  3552 E MainActivity: Caused by: java.security.cert.CertificateException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:08.982  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.verifyChain(TrustManagerImpl.java:674)
01-06 16:32:08.982  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.checkTrustedRecursive(TrustManagerImpl.java:551)
01-06 16:32:08.982  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.checkTrusted(TrustManagerImpl.java:507)
01-06 16:32:08.982  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.checkTrusted(TrustManagerImpl.java:426)
01-06 16:32:08.982  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.getTrustedChainForServer(TrustManagerImpl.java:354)
01-06 16:32:08.982  3552  3552 E MainActivity:  at android.security.net.config.NetworkSecurityTrustManager.checkServerTrusted(NetworkSecurityTrustManager.java:94)
01-06 16:32:08.982  3552  3552 E MainActivity:  at android.security.net.config.RootTrustManager.checkServerTrusted(RootTrustManager.java:89)
01-06 16:32:08.982  3552  3552 E MainActivity:  at com.android.org.conscrypt.Platform.checkServerTrusted(Platform.java:224)
01-06 16:32:08.982  3552  3552 E MainActivity:  at com.android.org.conscrypt.ConscryptFileDescriptorSocket.verifyCertificateChain(ConscryptFileDescriptorSocket.java:407)
01-06 16:32:08.982  3552  3552 E MainActivity:  at com.android.org.conscrypt.NativeCrypto.SSL_do_handshake(Native Method)
01-06 16:32:08.983  3552  3552 E MainActivity:  at com.android.org.conscrypt.NativeSsl.doHandshake(NativeSsl.java:387)
01-06 16:32:08.983  3552  3552 E MainActivity:  at com.android.org.conscrypt.ConscryptFileDescriptorSocket.startHandshake(ConscryptFileDescriptorSocket.java:226)
01-06 16:32:08.983  3552  3552 E MainActivity:  ... 26 more
01-06 16:32:08.983  3552  3552 E MainActivity: Caused by: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:08.983  3552  3552 E MainActivity:  ... 38 more
01-06 16:32:08.983  3552  3552 E MainActivity: HTTP status code: 500
01-06 16:32:08.985  3552  3552 E b.habdroid.bet: Invalid ID 0x00000000.
01-06 16:32:08.986  3552  3552 D ContentController: Indicate server failure (message Connection unsuccessful. An unexpected response was received while trying to connect to the configured openHAB server (HTTP code 500).)
01-06 16:32:10.197  3552  3552 I am_on_top_resumed_lost_called: [0,org.openhab.habdroid.ui.MainActivity,topStateChangedWhenResumed]
01-06 16:32:10.197  3552  3552 D MainActivity: onPause()
01-06 16:32:10.198  3552  3552 I am_on_paused_called: [0,org.openhab.habdroid.ui.MainActivity,performPause]
01-06 16:32:10.215  3552  3552 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@5ea778c
01-06 16:32:10.232  3552  3552 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
01-06 16:32:10.236  3552  3552 I am_on_create_called: [0,org.openhab.habdroid.ui.AboutActivity,performCreate]
01-06 16:32:10.242  3552  3552 I am_on_start_called: [0,org.openhab.habdroid.ui.AboutActivity,handleStartActivity]
01-06 16:32:10.242  3552  3552 I am_on_resume_called: [0,org.openhab.habdroid.ui.AboutActivity,RESUME_ACTIVITY]
01-06 16:32:10.244  3552  9101 D CloudMessagingHelper: Got exception: org.openhab.habdroid.util.HttpClient$HttpException: Unauthorized
01-06 16:32:10.247  3552  3552 I am_on_top_resumed_gained_called: [0,org.openhab.habdroid.ui.AboutActivity,topStateChangedWhenResumed]
01-06 16:32:10.708  3552  3552 D MainActivity: onStop()
01-06 16:32:10.708  3552  3552 D PageConnectionHolderFragment: onStop()
01-06 16:32:10.708  3552  3552 I am_on_stop_called: [0,org.openhab.habdroid.ui.MainActivity,STOP_ACTIVITY_ITEM]
01-06 16:32:10.708  3552  3552 D MainActivity: onSaveInstanceState()
01-06 16:32:12.603  3552  3552 I am_on_top_resumed_lost_called: [0,org.openhab.habdroid.ui.AboutActivity,topStateChangedWhenResumed]
01-06 16:32:12.605  3552  3552 I am_on_paused_called: [0,org.openhab.habdroid.ui.AboutActivity,performPause]
01-06 16:32:12.614  3552  3552 I am_on_restart_called: [0,org.openhab.habdroid.ui.MainActivity,performRestartActivity]
01-06 16:32:12.615  3552  3552 D MainActivity: onStart()
01-06 16:32:12.615  3552  3552 D PageConnectionHolderFragment: onStart(), started false
01-06 16:32:12.615  3552  3552 D MainActivity: onAvailableConnectionChanged()
01-06 16:32:12.616  3552  3552 D PageConnectionHolderFragment: updateActiveConnections: URL list [], connection org.openhab.habdroid.core.connection.DefaultConnection@8766f67
01-06 16:32:12.616  3552  3552 I am_on_start_called: [0,org.openhab.habdroid.ui.MainActivity,handleStartActivity]
01-06 16:32:12.616  3552  3552 D MainActivity: onResume()
01-06 16:32:12.628  3552  3552 I am_on_resume_called: [0,org.openhab.habdroid.ui.MainActivity,RESUME_ACTIVITY]
01-06 16:32:12.628  3552  3552 I am_on_top_resumed_gained_called: [0,org.openhab.habdroid.ui.MainActivity,topWhenResuming]
01-06 16:32:12.631  3552  3705 D AbstractConnection: Checking reachability of https://<openhab-local-address>:8443/
01-06 16:32:12.709  3552  3705 D AbstractConnection: Socket connected (attempt  0)
01-06 16:32:12.709  3552  3705 D ConnectionFactory: Connecting to local URL
01-06 16:32:12.822  3552  3552 E MainActivity: Error: org.openhab.habdroid.util.HttpClient$HttpException: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:12.822  3552  3552 E MainActivity: org.openhab.habdroid.util.HttpClient$HttpException: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:12.822  3552  3552 E MainActivity:  at org.openhab.habdroid.util.HttpClient$method$$inlined$suspendCancellableCoroutine$lambda$1.onFailure(HttpClient.kt:149)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.RealCall$AsyncCall.run(RealCall.kt:146)
01-06 16:32:12.822  3552  3552 E MainActivity:  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
01-06 16:32:12.822  3552  3552 E MainActivity:  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
01-06 16:32:12.822  3552  3552 E MainActivity:  at java.lang.Thread.run(Thread.java:919)
01-06 16:32:12.822  3552  3552 E MainActivity: Caused by: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.ConscryptFileDescriptorSocket.startHandshake(ConscryptFileDescriptorSocket.java:231)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.kt:350)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.kt:308)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.connection.RealConnection.connect(RealConnection.kt:182)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.kt:238)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.kt:111)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.kt:79)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.connection.Transmitter.newExchange$okhttp(Transmitter.kt:163)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:35)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:82)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:84)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:71)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.logging.HttpLoggingInterceptor.intercept(HttpLoggingInterceptor.kt:152)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.kt:194)
01-06 16:32:12.822  3552  3552 E MainActivity:  at okhttp3.RealCall$AsyncCall.run(RealCall.kt:138)
01-06 16:32:12.822  3552  3552 E MainActivity:  ... 3 more
01-06 16:32:12.822  3552  3552 E MainActivity: Caused by: java.security.cert.CertificateException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.verifyChain(TrustManagerImpl.java:674)
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.checkTrustedRecursive(TrustManagerImpl.java:551)
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.checkTrusted(TrustManagerImpl.java:507)
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.checkTrusted(TrustManagerImpl.java:426)
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.TrustManagerImpl.getTrustedChainForServer(TrustManagerImpl.java:354)
01-06 16:32:12.822  3552  3552 E MainActivity:  at android.security.net.config.NetworkSecurityTrustManager.checkServerTrusted(NetworkSecurityTrustManager.java:94)
01-06 16:32:12.822  3552  3552 E MainActivity:  at android.security.net.config.RootTrustManager.checkServerTrusted(RootTrustManager.java:89)
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.Platform.checkServerTrusted(Platform.java:224)
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.ConscryptFileDescriptorSocket.verifyCertificateChain(ConscryptFileDescriptorSocket.java:407)
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.NativeCrypto.SSL_do_handshake(Native Method)
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.NativeSsl.doHandshake(NativeSsl.java:387)
01-06 16:32:12.822  3552  3552 E MainActivity:  at com.android.org.conscrypt.ConscryptFileDescriptorSocket.startHandshake(ConscryptFileDescriptorSocket.java:226)
01-06 16:32:12.822  3552  3552 E MainActivity:  ... 26 more
01-06 16:32:12.822  3552  3552 E MainActivity: Caused by: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
01-06 16:32:12.822  3552  3552 E MainActivity:  ... 38 more
01-06 16:32:12.822  3552  3552 E MainActivity: HTTP status code: 500
01-06 16:32:12.822  3552  3552 E b.habdroid.bet: Invalid ID 0x00000000.
01-06 16:32:12.822  3552  3552 D ContentController: Indicate server failure (message Connection unsuccessful. An unexpected response was received while trying to connect to the configured openHAB server (HTTP code 500).)
01-06 16:32:13.066  3552  3552 I am_on_stop_called: [0,org.openhab.habdroid.ui.AboutActivity,LIFECYCLER_STOP_ACTIVITY]
01-06 16:32:13.068  3552  3552 I am_on_destroy_called: [0,org.openhab.habdroid.ui.AboutActivity,performDestroy]
01-06 16:32:13.706  3552  3552 D MainActivity: onOptionsItemSelected()
01-06 16:32:14.607  3552  3552 I am_on_top_resumed_lost_called: [0,org.openhab.habdroid.ui.MainActivity,topStateChangedWhenResumed]
01-06 16:32:14.608  3552  3552 D MainActivity: onPause()
01-06 16:32:14.608  3552  3552 I am_on_paused_called: [0,org.openhab.habdroid.ui.MainActivity,performPause]
01-06 16:32:14.624  3552  3552 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@ca63ec4
01-06 16:32:14.636  3552  3552 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
01-06 16:32:14.644  3552  3552 I am_on_create_called: [0,org.openhab.habdroid.ui.PreferencesActivity,performCreate]
01-06 16:32:14.714  3552  3552 D Ringtone: Successfully created local player
01-06 16:32:14.856  3552  3552 D PreferencesActivity: Removing notification prefs for < 25
01-06 16:32:14.873  3552  3552 I am_on_start_called: [0,org.openhab.habdroid.ui.PreferencesActivity,handleStartActivity]
01-06 16:32:14.873  3552  3552 I am_on_resume_called: [0,org.openhab.habdroid.ui.PreferencesActivity,RESUME_ACTIVITY]
01-06 16:32:14.880  3552  3552 I am_on_top_resumed_gained_called: [0,org.openhab.habdroid.ui.PreferencesActivity,topStateChangedWhenResumed]
01-06 16:32:14.890  3552  3566 V MediaPlayer: resetDrmState:  mDrmInfo=null mDrmProvisioningThread=null mPrepareDrmInProgress=false mActiveDrmScheme=false
01-06 16:32:14.890  3552  3566 V MediaPlayer: cleanDrmObj: mDrmObj=null mDrmSessionId=null
01-06 16:32:14.969  3552  3552 W MediaPlayer: mediaplayer went away with unhandled events
01-06 16:32:15.385  3552  3552 D MainActivity: onStop()
01-06 16:32:15.386  3552  3552 D PageConnectionHolderFragment: onStop()
01-06 16:32:15.386  3552  3552 I am_on_stop_called: [0,org.openhab.habdroid.ui.MainActivity,STOP_ACTIVITY_ITEM]
01-06 16:32:15.386  3552  3552 D MainActivity: onSaveInstanceState()
01-06 16:32:16.683  3552  3552 W ListPreference: Setting a summary with a String formatting marker is no longer supported. You should use a SummaryProvider instead.
01-06 16:32:17.960  3552  3552 I am_on_top_resumed_lost_called: [0,org.openhab.habdroid.ui.PreferencesActivity,topStateChangedWhenResumed]
01-06 16:32:17.961  3552  3552 I am_on_paused_called: [0,org.openhab.habdroid.ui.PreferencesActivity,performPause]
01-06 16:32:17.968  3552  3552 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@121c315
01-06 16:32:17.987  3552  3552 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
01-06 16:32:17.989  3552  3552 I FloatingActionButton: Setting a custom background is not supported.
01-06 16:32:18.007  3552  3552 I am_on_create_called: [0,org.openhab.habdroid.ui.LogActivity,performCreate]
01-06 16:32:18.009  3552  3552 I am_on_start_called: [0,org.openhab.habdroid.ui.LogActivity,handleStartActivity]
01-06 16:32:18.009  3552  3552 I am_on_resume_called: [0,org.openhab.habdroid.ui.LogActivity,RESUME_ACTIVITY]
01-06 16:32:18.013  3552  3552 I am_on_top_resumed_gained_called: [0,org.openhab.habdroid.ui.LogActivity,topStateChangedWhenResumed]
01-06 16:32:18.035  3552  3552 D LogActivity: onCreateOptionsMenu()
```

</details>


#### openHAB Server log

I don't see anything in there.

**Comment by mueller-ma** [comment]
> org.openhab.habdroid.util.HttpClient$HttpException: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.

Besides the certificate issue the wrong error message is shown.
This sounds like an issue with MTM and okhttp 4.

**Comment by mueller-ma** [comment]
There was a bug with okhttp and custom trust manager (like MTM) on Android 10. The [fix](https://github.com/square/okhttp/commit/ed9af27531b02f5ef334b7d606ef35c66f541958) is in 2.11.5-beta.

Can you try it again after updating?

**Comment by openhab-5iver** [comment]
I'll report back as soon as I see it in the PlayStore.

**Comment by openhab-5iver** [comment]
I'm not getting the 500 error in 2.11.5-beta. Thank you!

**Comment by mueller-ma** [comment]
Fixed by #1764
